cmd/kube-proxy/app/server.go

```diff
@@ -235,7 +235,7 @@ func NewOptions() *Options {
 }
 
 // Complete completes all the required options.
-func (o *Options) Complete() error {
+func (o *Options) Complete(fs *pflag.FlagSet) error {
```

Comment:
should we make the Flags part of the Options struct instead of plumbing it down?
I see kube-scheduler has them there too, it feels like is cleaner

https://github.com/kubernetes/kubernetes/blob/a287fd6aa6aae212a65ca415367cfccb70cd6190/cmd/kube-scheduler/app/options/options.go#L75-L76

Reply:
sgtm